### DISCUSSION TITLE
Help：how to config lazyterm in edgy.nvim ?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#3688] [Q&A] [answered]
I want to be able to execute some commands automatically when I open lazyterm. How should I configure it?

```lua
return {
  {
    "folke/edgy.nvim",
    opts = function()
      local opts = {
        bottom = {
          {
            ft = "lazyterm",
            title = "LazyTerm",
            size = { width = 0.5, height = 0.4 },
            filter = function(buf)
              return not vim.b[buf].lazyterm_cmd
            end,
          },
          {
            ft = "lazyterm",
            title = "btop",
            size = { width = 0.5, height = 0.4 },
            open = function()
              -- run btop command
            end,
            filter = function(buf)
              return not vim.b[buf].lazyterm_cmd
            end,
          },

-- omit
```

**Comment by dpetka2001** [comment]
@iamxiaojianzheng So running `LazyVim.terminal("btop")` seems to open a terminal window with `btop`. Can you try integrating that to your edgy configuration?

**Reply by iamxiaojianzheng** [reply]
Strangely enough, I managed to open it through the terminal. 

But the effect is a little bit wrong, there seems to be a layer. It's not quite the same as when I open terminal and then type in the btop command.

**Reply by dpetka2001** [reply, marked as answer]
Try `LazyVim.terminal("btop", { backdrop = 100 })`.

**Reply by dpetka2001** [reply]
@iamxiaojianzheng Did you try this? Does it do what you expect to see?

**Reply by iamxiaojianzheng** [reply]
You are right !!! Worship, worship

**Comment by iamxiaojianzheng** [comment]


**Reply by iamxiaojianzheng** [reply]
@dpetka2001  I suspect it may be the floating effect inside terminal

**Reply by dpetka2001** [reply]
See my other comment and try that to see if it changes anything.
